templates/registry/registry-dpl.yaml

```diff
@@ -21,6 +21,11 @@ spec:
         checksum/secret: {{ include (print $.Template.BasePath "/registry/registry-secret.yaml") . | sha256sum }}
         checksum/secret-jobservice: {{ include (print $.Template.BasePath "/jobservice/jobservice-secrets.yaml") . | sha256sum }}
         checksum/secret-core: {{ include (print $.Template.BasePath "/core/core-secret.yaml") . | sha256sum }}
+{{- if .Values.registry.enablePrometheus }}
+        prometheus.io/path: "/metrics"
+        prometheus.io/port: "5001"
+        prometheus.io/scrape: "true"
```

Comment:
What do you expect to consume these annotations? I did some searching and it looks like `prometheus.io/scrape` used to be present in some example configurations, but it was removed in https://github.com/prometheus/prometheus/pull/4131 because people copied it literally rather than using it as inspiration.

(From our point of view, I expect to be using prometheus-operator's `ServiceMonitor` to configure what to scrape, so I'm pretty agnostic to these annotations).

Reply:
hmm, good point -- I must have copied these from those examples. I'll remove the annotations from the PR as different configurations will configure scraping in different ways.